.gitignore

```diff
@@ -247,4 +247,5 @@ ENV/
 env.bak/
 venv.bak/
 
-**/*secrets*
\ No newline at end of file
+**/*secrets*
+*.data/
```

Comment:
if you make this `**/*.data` it will also look in all subfolders

Reply:
Okay great done.